float64 percentage
---
bool success
